Mono.Debugging/Mono.Debugging.Client/BreakpointStore.cs

```diff
@@ -127,20 +127,24 @@ void ICollection<BreakEvent>.Add (BreakEvent bp)
 			Add (bp);
 		}
 		
-		public bool Add (BreakEvent bp)
+		public bool Add (BreakEvent be)
 		{
-			if (bp == null)
-				throw new ArgumentNullException (nameof (bp));
+			if (be == null)
+				throw new ArgumentNullException (nameof (be));
 
 			if (IsReadOnly)
 				return false;
 
+			if (be is Breakpoint bp) {
+				bp.SetFileName (Path.GetFullPath(bp.FileName)); // TODO what happens if there is no filepath?
```

Comment:
Should this be using the ResolveFullPath method? That resolves symlinks when not on Windows.

https://github.com/mono/monodevelop/blob/ca84d5037376cea6efc342e101d72966589ef4ed/main/src/core/MonoDevelop.Core/MonoDevelop.Core/FilePath.cs#L89

Reply:
Yep, we most likely want to resolve symlinks here.

Reply:
ResolveFullPath was used before in this case.

Reply:
done!